src/sonic-yang-mgmt/tests/test_cfghelp.py

```diff
@@ -0,0 +1,170 @@
++---------+-------------------------------------------+-------------+-----------+-------------+
+
+"""
+
+dscp_to_tc_table_field_output="""\
+
+DSCP_TO_TC_MAP
+Description: DSCP_TO_TC_MAP part of config_db.json
+
+key - name
++---------+------------------------------------------------------+-------------+-----------+-------------+
+| Field   | Description                                          | Mandatory   | Default   | Reference   |
++=========+======================================================+=============+===========+=============+
+| name    |                                                      |             |           |             |
++---------+------------------------------------------------------+-------------+-----------+-------------+
+| dscp    | This field is for storing mapping between two fields |             |           |             |
++---------+------------------------------------------------------+-------------+-----------+-------------+
+| tc      | This field is for storing mapping between two fields |             |           |             |
++---------+------------------------------------------------------+-------------+-----------+-------------+
+
+"""
+
+acl_rule_table_field_output="""\
+
+ACL_RULE
+Description: ACL_RULE part of config_db.json
+
+key - ACL_TABLE_NAME:RULE_NAME
++-----------+-----------------------------------------+-------------+-----------+-------------+
+| Field     | Description                             | Mandatory   | Default   | Reference   |
++===========+=========================================+=============+===========+=============+
+| ICMP_TYPE | Mutually exclusive in group icmp        |             |           |             |
+|           | when IP_TYPE in ANY,IP,IPV4,IPv4ANY,ARP |             |           |             |
++-----------+-----------------------------------------+-------------+-----------+-------------+
+
+"""
+
+class TestCfgHelp(TestCase):
+
+    def setUp(self):
+        self.test_dir = os.path.dirname(os.path.realpath(__file__))
+        self.script_file = os.path.join(self.test_dir, '..', 'sonic-cfg-help')
+
+    def run_script(self, argument):
+        print('\n    Running sonic-cfg-help ' + argument)
+        output = subprocess.check_output(self.script_file + ' ' + argument, shell=True)
+
+        output = output.decode()
+
+        linecount = output.strip().count('\n')
+        if linecount <= 0:
+            print('    Output: ' + output.strip())
+        else:
+            print('    Output: ({0} lines, {1} bytes)'.format(linecount + 1, len(output)))
+        return output
+
+    def test_dummy_run(self):
+        argument = ''
+        output = self.run_script(argument)
+        self.assertEqual(output, output1)
+
+    def test_single_table(self):
+        argument = '-t AUTO_TECHSUPPORT'
+        output = self.run_script(argument)
+        self.assertEqual(output, techsupport_table_output)
```

Comment:
If we modify techsupport yang model, this unit test will fail?

Reply:
Maybe we can create some dummy yang models for unit test.
And then we can modify Yang model, and don't need to update this unit test.

Reply:
Hi @ganglyu ,
I prefer to use any active yang model rather than introducing a dummy for the test. We do have instances of such tests in other repositories where if schema is changed, the UT needs to be updated.
I don't think the yang model I chosen would change quite often. Even if someone changes it and accidentally misses to update UT, it will be caught during build time and build will fail, forcing the user to modify the test.